Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/resource/VS_VERSION_CHILD.java

```diff
@@ -185,8 +185,8 @@ public boolean valueIsUnicodeString() {
 
 	/**
 	 * @return true if value is 4-byte integer value in memory 
-	 * while string value return by {@link #getValue()} is a
-	 * numeric hex string.
+	 * while string value return by {@link DataType#getValue(ghidra.program.model.mem.MemBuffer, ghidra.docking.settings.Settings, int)
+	 * DataType.getValue(MemBuffer, Settings, int)} is a numeric hex string.
```

Comment:
I think this is what was intended or maybe Data#getValue()? This one was difficult to determine.